app/src/main/java/org/mozilla/fenix/library/history/viewholders/HistoryListItemViewHolder.kt

```diff
@@ -43,12 +48,13 @@ class HistoryListItemViewHolder(
         }
     }
 
+    @Suppress("UndocumentedPublicFunction")
```

Comment:
I don't see anything wrong with the @Suppress. The default value wasn't ever used, removing it will improve the code base. And the `[example](https://searchfox.org/mozilla-mobile/source/android-components/components/feature/prompts/src/main/java/mozilla/components/feature/prompts/creditcard/CreditCardItemViewHolder.kt#29)` documentation adds zero value in understanding the code. Comments should provide explanation of a higher abstraction layer than the code itself. 
A comment like `Binds the view with the provided [CreditCard].` inside a ViewHolder class on a bind method doesn't add much really.

Reply:
Sorry, we aren't moving forward with this until we address this. I will even allow you to add this to the detekt-baseline, but since there's no instances of `@Suppress("UndocumentedPublicFunction")` anywhere in our codebase https://searchfox.org/mozilla-mobile/search?q=UndocumentedPublicFunction&path=, I see no reason why we should add this exception here.